package.json

```diff
@@ -16,6 +16,7 @@
     "tsc": "tsc -b",
     "watch": "tsc -b -w",
     "eslint": "lerna exec --stream --no-bail -- eslint src/**/*.ts",
+    "eslint-fix": "yarn -s eslint --fix",
```

Comment:
Why is this needed? You can just do `yarn eslint --fix` and it's only 1 or 2 extra characters. This is how all our repos are setup so I'm not keen on adding a custom command for this :shrug:

Reply:
maybe not everyone knows eslint can fix things for them 🤷‍♂️

Reply:
Maybe we should add something to the Contributing docs section instead then?

Reply:
Done, see what you think

Reply:
LGTM :+1: